app/buildpacks/node_buildpack.rb

```diff
@@ -5,6 +5,8 @@ class NodeBuildpack
   SUPPORTED = %w(5 6 8 10).freeze
   TEMPLATE  = [
     'FROM <%= base_images_registry_url %>/node:<%= node_version %>',
+    'ENV DATADOG_LOGS_SOURCE nodejs',
```

Comment:
Note that in our current cluster, `busbar apps | grep node` returns a bunch of microservices denoted as `(node)` and not `(nodejs)`. The same is true for the name of this buildpack -- `node_buildpack` as opposed to `nodejs_buildpack`.

To keep things consistent, I like to create an enum for all magic strings that are used throughout the application (e.g., something like `const NODE = 'node'`) in a global config and then use write like `'ENV DATADOG_LOGS_SOURCE <%= NODE %>'` on this line.

Reply:
Datadog AFAIK labels the language as `nodejs` and not `node`.

Reply:
So not only are we hardcoding "Datadog" into this repo, we're also creating a hard dependency on how that third-party service classifies Node.js projects (by making sure that this code says `nodejs` as opposed to `node` because that's how it's classified on Datadog)?